### PR TITLE
Add `README.md`.

### LINKED ISSUE #3: Add README content.

### PR DESCRIPTION
**Pull request by JSAssassin**
closes #3

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Thanks!

**Review by mattcollier — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+### decodeBits({encoded})
+Decodes and uncompresses a given a base64url encoded string to a Uint8Array.
+```js
+await Bitstring.decodeBits({encoded});
+```
+
+### compressBits()
+Compresses bits to a Uint8Array.
+```js
+await bitstring.compressBits();
```
Comment (aljones15) [inline README.md:63]: I might be wrong here, but it looks like compressBits gzips a Uint8array:

https://github.com/digitalbazaar/bitstring/blob/b11fb56cecb96cd00b097ef709726f65423bcef0/main.js#L55-L57

**Review by aljones15 — APPROVED** [review]
This looks good to me, but do double check that compressBits documentation.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
-# bitstring
-A Bitstring module for universal JavaScript
+# Bitstring
+A Bitstring module for universal JavaScript.
+
+## Install
+
+To install get the latest version from NPM
```
Comment (davidlehn) [inline README.md:6]:
```suggestion
To install get the latest version from NPM:
```

File: README.md (hunk 1 above)
Comment (JSAssassin) [reply README.md:63]: @aljones15 yeah, you're right `this.bits` is Uint8Array but the compressed output is also Uint8Array. So, I think this is fine.

File: README.md (hunk 2 above)
Comment (JSAssassin) [reply README.md]: Updated b6a9f2e

**Review by davidlehn — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (aljones15) [reply README.md:63]: ok did not know the gzip operation produced a Uint8Array, but that does makes sense. Don't let this comment hold you up feel to merge as you have multiple approvals.

**Pull request merged**